core/trino-main/src/test/java/io/trino/sql/query/TestColumnMask.java

```diff
@@ -212,7 +212,8 @@ public void testMultipleMasksOnSameColumn()
                 USER,
                 new ViewExpression(USER, Optional.empty(), Optional.empty(), "custkey * 2"));
 
-        assertThat(assertions.query("SELECT custkey FROM orders WHERE orderkey = 1")).matches("VALUES BIGINT '-740'");
+        // When there are multiple masks on the same column, the latter one overrides the previous ones
+        assertThat(assertions.query("SELECT custkey FROM orders WHERE orderkey = 1")).matches("VALUES BIGINT '740'");
```

Comment:
Shouldn't this now fail with `.hasMessageMatching("Multiple masks on a single column is not supported: column");`?

Reply:
That only happens after the second commit. Perhaps they should be reordered, so that the prohibition on multiple masks goes first.

Reply:
OTOH, it does pass in the CI apparently

Reply:
Ah, I see - the test is using `TestingAccessControlManager`, which replaces the actual `AccessControlManager` and does not check for multiple masks.